src/dom.js

```diff
@@ -408,7 +408,9 @@ export function getDataParamsFromAttributes(element, opt_computeParamNameFunc) {
   const params = Object.create(null);
   for (let i = 0; i < attributes.length; i++) {
     const attr = attributes[i];
-    const matches = attr.name.match(/^data-param-(.+)/);
+    const attrName = attr.name;
+    const matches = attrName.match(/^data-param-(.+)/) ||
+      attrName.match(/^data-vars-(.+)/);
```

Comment:
I don't think this will work. This changes the behavior for existing pages/clients. I think we should add another parameter to the function that accepts a regex called  `opt_paramPattern`. If the value is not provided, you can default to `/^data-param-(.+)/`. Then from instrumentation.js, pass in the value of the pattern.